### PR TITLE
Refactor of including javascript content to brightcove player

### PR DESCRIPTION
**Pull request by sendr**

### CODE REVIEW

File: video_xblock/static/js/videojs-tabindex.js [hunk 1]
```diff
@@ -12,33 +12,41 @@ domReady(function() {
...
+            controlBar = _player.controlBar;
+        };
+        var controlBarActions = Object.keys(controlBar);
         var controlsTabOrder = [
             'progressControl',
             'playToggle',
             'playbackRateMenuButton',
             'volumeMenuButton',
-            'resolutionSwitcher',
+//            'resolutionSwitcher',
```
Comment (z4y4ts) [inline video_xblock/static/js/videojs-tabindex.js]: Delete it

File: video_xblock/static/js/videojs-tabindex.js [hunk 2]
```diff
@@ -12,33 +12,41 @@ domReady(function() {
...
-            'volumeMenuButton': _player.controlBar.childNameIndex_.volumeMenuButton.volumeBar.el_,
-            'fullscreenToggle': _player.controlBar.childNameIndex_.fullscreenToggle.el_,
-            'resolutionSwitcher': _player.controlBar.resolutionSwitcher,
-            'playbackRateMenuButton': _player.controlBar.childNameIndex_.playbackRateMenuButton.el_
+            'progressControl': controlBar.progressControl.seekBar.el_,
+            'playToggle': controlBar.playToggle.el_,
+            'captionsButton': controlBar.captionsButton.el_,
+            'volumeMenuButton': controlBar.volumeMenuButton.volumeBar.el_,
+            'fullscreenToggle': controlBar.fullscreenToggle.el_,
+//            'resolutionSwitcher': _player.controlBar.resolutionSwitcher,
```
Comment (z4y4ts) [inline video_xblock/static/js/videojs-tabindex.js]: Delete it

File: video_xblock/static/js/videojs-tabindex.js [hunk 3]
```diff
@@ -12,33 +12,41 @@ domReady(function() {
...
+            'playbackRateMenuButton': controlBar.playbackRateMenuButton.el_
           };
 
         // Swith off tabIndex for volumeMenuButton and free slot for volumeBar
```
Comment (z4y4ts) [inline video_xblock/static/js/videojs-tabindex.js]: Typo: Switch

File: video_xblock/static/css/brightcove.css [hunk 4]
```diff
@@ -396,3 +396,42 @@ body .video-js .vjs-menu-button-popup .vjs-menu .vjs-menu-item:hover {
   background-color: #171a1b;
   color: #0ea6ec;
 }
+
```
Comment (z4y4ts) [inline video_xblock/static/css/brightcove.css]: Extract this new code to `.../css/transcripts.css`.

File: video_xblock/backends/brightcove.py [hunk 5]
```diff
@@ -32,28 +32,40 @@ def get_frag(self, **context):
...
+            self.add_js_content('../static/js/brightcove-videojs-init.js', **context)
         )
         frag.add_css(self.resource_string(
             '../static/css/brightcove.css'
         ))
         return frag
+
+    def add_js_content(self, path, **context):
+        """
+        Helper for adding javascript content to brightcove player
```
Comment (polesye) [inline video_xblock/backends/brightcove.py]: Is it only for brightcove or can be used in other players?

Comment (z4y4ts) [inline video_xblock/backends/brightcove.py]: Right now we need this for Brightcove only. But it can be used for other players too.

Comment (polesye) [inline video_xblock/backends/brightcove.py]: In this case, it makes sense to avoid using `brightcove` player name in the docstring

**Pull request merged**